<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInWindow</class>
 <widget class="LogInWindow" name="LogInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>515</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LogInWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>501</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>421</width>
      <height>241</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLineEdit {
	border: 0px
	border-bottom: 1px solid black
}</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>321</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pasword</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="password_line">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="login_line"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="sign_in_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign in</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginErrorLine">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border: 0px;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>200</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color : white;
	border: 0px;
	color: blue;
	text-decoration: underline;
}</string>
     </property>
     <property name="text">
      <string>Use Event Code</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogInWindow</class>
   <extends>QWidget</extends>
   <header>loginwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
